wrappers/preseq/estimated_coverage/wrapper.py

```diff
@@ -3,11 +3,24 @@
 __email__ = "behram.radmanesh@nih.gov"
 __license__ = "MIT"
 
+import os
+
 # import snakemake's ability to execute shell commands
 from snakemake.shell import shell
 
+# specify an output directory from snakemake otherwise insert default
+outdir = os.path.dirname(snakemake.output[0])
+if not outdir:
+    outdir = '.'
+
+try:
+    output = snakemake.params.output
+except AttributeError:
+    output = ""
+
+
 # execute bam2mr
 shell("bam2mr {snakemake.input[0]} > sorted.nr")
 
 # execute preseq gc_extrap
-shell("preseq gc_extrap {snakemake.params} {snakemake.output[0]} sorted.nr")
+shell("preseq gc_extrap {snakemake.params.output} {snakemake.output[0]} sorted.nr")
```

Comment:
How about deleting the sorted.nr afterwards? Is there any reason to keep it around?